server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/service/ClearMailRepositoryTask.java

```diff
@@ -87,10 +88,9 @@ public String type() {
     }
 
     public long getRemainingSize() {
-        try {
-            return mailRepository.size();
-        } catch (MessagingException e) {
-            throw new RuntimeException(e);
-        }
+        return mailRepositories
+                .stream()
+                .map(Throwing.function(MailRepository::size).sneakyThrow())
+                .reduce(0L, (a, b) -> a + b);
```

Comment:
I suggest a more readable:

```
return mailRepository.stream()
    .mapToInt(Throwing.function(MailRepository::size).sneakyThrow())
    .sum();
```

Reply:
We get a `Long`, the compiler can't cast it.

Reply:
consider:
```java
        return mailRepositories
                .stream()
                .mapToLong(Throwing.toLongFunction(MailRepository::size).sneakyThrow())
                .sum();
```